docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc

```diff
@@ -0,0 +1,21 @@
+[id='con-introducing-raincatcher-{chapter}']
+= Introducing {WFM-RC-NameLong}
+
+//downstream == {WFM-RC-NameLong} is a Workforce Management Platform and is also referred to as {WFM-RC-NameShort}.
+{WFM-RC-NameLong} is a Workforce Management Platform.
+The latest version of {WFM-RC-NameShort} is {WFM-RC-Version}.
+
+The primary goal of {WFM-RC-NameShort} is to deliver a set of modules and a user-interface implementation for developers to integrate {WFM-RC-NameShort} features into their application.
```

Comment:
goal is to deliver modules to integrate them. Repetition of product name is strange here.